doc/reference/lora_lorawan/index.rst

```diff
@@ -1,26 +1,53 @@
-.. _lorawan_api:
+.. _lora_lorawan_api:
 
-LoRaWAN
-#######
+LoRa and LoRaWAN
+################
 
 Overview
 ********
 
-Long Range Wide Area Network (LoRaWAN) is a network layer protocol based on
-the LoRa PHY.
+LoRa (abbrev. for Long Range) is a proprietary low-power wireless
+communication protocol developed by the `Semtech Corporation`_.
+
+LoRa defines the physical layer of the protocol stack, which uses
+chirp-spread-spectrum (CSS) modulation.
+
+LoRaWAN (for Long Range Wide Area Network) defines a networking layer
+on top of the LoRa PHY.
+
+Zephyr provides APIs for LoRa to send raw data packets directly over the
+wireless interface as well as APIs for LoRaWAN to interact with a gateway
```

Comment:
Zephyr provides APIs for LoRa to send raw data packets to other LoRa devices forming the Point-to-Point (P2P) communication.

Reply:
Zephyr also provides APIs for LoRaWAN to connect the end device to the internet through a gateway.

Reply:
LoRa is not necessarily only point-to-point communication, you can also broadcast messages to multiple nodes. I changed the wording for the LoRaWAN part but would like to keep the first part as is.